src/key_variants/double_key.rs

```diff
@@ -78,19 +90,17 @@ impl Serializable<64> for PublicKeyPair {
 #[derive(Clone, Copy, Debug)]
 #[cfg_attr(feature = "canon", derive(Canon))]
 pub struct Proof {
-    U: JubJubScalar,
-    R: PublicKeyPair,
+    u: JubJubScalar,
+    public: PublicKeyPair,
```

Comment:
If you want it short, I will call it `keys`, or `key_pair`, otherwise you can have a more verbose `public_key_pair`; but `public` doesn't tell much IMVHO.

Reply:
Fixed, calling it `keys`